config/spotbugs/baseline.xml

```diff
@@ -0,0 +1,1061 @@
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\SingleAccountPublicClientApplication$4.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\SingleAccountPublicClientApplication$5.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\SingleAccountPublicClientApplication.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TenantProfile.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenParameters$Builder$1.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenParameters$Builder.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenParameters.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenParametersAdapter.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenShareResult$TokenShareExportFormat.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\TokenShareResult.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\client\WebViewSelection.class</Jar>
+    <Jar>C:\temp\android_auth\msal\intermediates\javac\localDebug\classes\com\microsoft\identity\msal\BuildConfig.class</Jar>
+    <AuxClasspathEntry>C:\temp\android_auth\msal\intermediates\compile_r_class_jar\localDebug\R.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\b6fbbdadfcff642127829062bce22562\appcompat-1.0.2-api.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\3658c0f0705bb2e38e851f278ccc95f8\browser-1.0.0-api.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\0be1db90cc4ed622b4eef576769e0d0c\jetified-gson-2.8.5.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\234b7560de0f7ee089e2873186f07be2\jetified-nimbus-jose-jwt-8.2.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\be6c6301159f0ceed84e44e98562c597\constraintlayout-1.1.3-api.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\temp\android_auth\common\intermediates\compile_library_classes_jar\debug\classes.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\8c150e30907e6dae2c2a9bca9443bae6\fragment-1.0.0-api.jar</AuxClasspathEntry>
```

Comment:
Mentioned via Teams but just raising awareness - there are machine specific paths in here

Reply:
I got rid of those.